src_c/scrap.c

```diff
@@ -222,7 +222,12 @@ _scrap_get_scrap(PyObject *self, PyObject *args)
                 val = PyDict_GetItemString(_clipdata, scrap_type);
```

Comment:
Should `PyDict_GetItemWithError()` be used just to be safe?

Reply:
Yes, good idea.